.jshintrc

```diff
@@ -10,7 +10,7 @@
     "latedef": "nofunc",
     "laxbreak": true,
     "maxdepth": 4,
-    "maxparams": 5,
+    "maxparams": 6,
```

Comment:
Should I disable the jshint rule for that line or should I modify the rule for this file?
/* jshint maxparams: 6 */
versus
// jshint ignore:line